src/workerd/jsg/compile-cache.c++

```diff
@@ -28,4 +34,21 @@ kj::Maybe<CompileCache::Data&> CompileCache::find(kj::StringPtr key) const {
   return kj::none;
 }
 
+void CompileCache::serialize(capnp::MessageBuilder& message) const {
+  capnp::DynamicStruct::Builder builder = message.initRoot<workerd::tools::CompileCache>();
```

Comment:
Why are you using the capnp dynamic API here? You've included the generated-code header so why not use it?

Reply:
You're absolutely right. Thank you for the review.